twine/settings.py

```diff
@@ -23,6 +25,16 @@
 from twine import utils
 
 
+def _setup_logging(verbose: bool) -> None:
+    """Initialize a logger based on verbosity available throughout twine."""
+    logger = logging.getLogger("twine")
+    log_level = logging.INFO if verbose else logging.WARNING
+    handler = logging.StreamHandler(sys.stdout)
+    handler.setLevel(log_level)
+    logger.addHandler(handler)
+    logger.setLevel(log_level)
```

Comment:
Echoing https://github.com/pypa/twine/pull/669#discussion_r446510744, I'm pretty sure we only need one call to `setLevel`. And, for now, maybe the intermediate variables aren't necessary? But that's just a style comment; it's fine to leave them in.

```suggestion
    root_logger.addHandler(logging.StreamHandler(sys.stdout))
    root_logger.setLevel(logging.INFO if verbose else logging.WARNING)
```